### DISCUSSION TITLE
Backend.ai logging level.

### DISCUSSION THREAD

**Discussion by mindgitrwx** [lablup/contribution-academy-2021#93] [Q&A]
Let's talk about the logging level.

**Additional context**

Check out this link:
https://sematext.com/blog/logging-levels/

The picture shows shells on backend.ai debug mode execution. 
![](https://imgur.com/Bc3Qia4.png)

**Comment by lizable** [comment]
@mindgitrwx  I wonder what you want to ask about the logging level. :) Could you plz tell us in detail?
